srcpkgs/ffmpeg/template

```diff
@@ -198,9 +178,49 @@ ffmpeg-devel_package() {
 		libavfilter>=${version}_${revision}
 		libpostproc>=${version}_${revision}
 		libswscale>=${version}_${revision}
-		libswresample>=${version}_${revision}
-		libavresample>=${version}_${revision}"
+		libswresample>=${version}_${revision}"
 	short_desc+=" - development files"
+	alternatives="
```

Comment:
I think you should just remove the manpages and let -devel packages conflict. This will only cause issues.